wofs/filters.py

```diff
@@ -92,8 +92,7 @@ def c2_filter(pq):
     """
     
     masking = np.zeros(pq.shape, dtype=np.uint8)
-    masking[((pq & C2_DILATED_BITS)).astype(np.bool)] += constants.MASKED_CLOUD
-    masking[((pq & C2_CLOUD_BITS)).astype(np.bool)] += constants.MASKED_CLOUD
+    masking[((pq & C2_DILATED_BITS)).astype(np.bool) | ((pq & C2_CLOUD_BITS)).astype(np.bool) | ((pq & C2_CIRRUS_BITS)).astype(np.bool)] += constants.MASKED_CLOUD
```

Comment:
This line is too long, can you please split it over three lines? Maybe just use `black` or some formatter to automatically do it :-)